oidc-ui/public/locales/ar.json

```diff
@@ -74,12 +74,15 @@
   "LoginQRCode": {
     "sign_in_with_inji": "تسجيل الدخول باستخدام Inji",
     "scan_with_inji": "امسح باستخدام تطبيق Inji لتسجيل الدخول",
```

Comment:
inji specific keywords should be removed now

Reply:
resolved in this [c70d6c0](https://github.com/mosip/esignet/pull/367/commits/c70d6c0aeeae68638f5799c8a92ef599a29380be)